activerecord/lib/active_record/associations.rb

```diff
@@ -71,6 +71,19 @@ def initialize(reflection = nil)
     end
   end
 
+  class InverseOfAssociationNotBidirectionalError < ActiveRecordError # :nodoc:
+    attr_reader :reflection
+    def initialize(reflection = nil)
+      @reflection = reflection
+
+      if reflection
+        super("Inverse association #{reflection.name} (#{reflection.inverse_of.name.inspect} in #{reflection.class_name}) is not bi-directional.")
+      else
+        super("Inverse association is not bi-directional.")
```

Comment:
Even I not sure what this error message means. It seems like it is a requirement for inverse_of to be by directional, but I don't think this is the problem here, is it? 

The problem is that the association on the other model doesn't exist, so we should be explicit about that.

Something like:

```
Inverse association doesn't not exist in the associated class
```

Reply:
@rafaelfranca  I have corrected the message you commented. 🙏 
Should I L81 have also been changed?